.travis.yml

```diff
@@ -64,7 +64,8 @@ jobs:
         - java --version
       install: skip
       script:
-        - ./.travis/setup_database &
```

Comment:
The removal of the background task `&` is intentional, we really want the database pieces to complete before http-server starts. The server does abort-launch if DB is not available.